src/pages/builder/Builder.jsx

```diff
@@ -1,3 +1,170 @@
+import GjsEditor from '@grapesjs/react';
+import gjsForms from 'grapesjs-plugin-forms';
+import './customStyles.module.css'
+
+const gjsOptions = {
```

Comment:
let's please create a utils folder inside the src directory src/utils and a gjsOptions file inside it where we can declare and export the object. 🙏🏼